<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageRecord</class>
 <widget class="QWidget" name="MessageRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="all">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#all:checked
{
border:none;
background-color: rgb(20, 153, 255);
color:white;
border-radius:2px;
}
QPushButton#all
{
border:none;
background-color: transparent;
color:black;
}
QPushButton#all:hover
{
background-color: rgb(20, 153, 255);
}</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="photo">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>56</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#photo:checked
{
border:none;
background-color: rgb(20, 153, 255);
color:white;
border-radius:2px;
}
QPushButton#photo
{
border:none;
background-color: transparent;
color:black;
}
QPushButton#photo:hover
{
background-color: rgb(20, 153, 255);
}</string>
       </property>
       <property name="text">
        <string>Photo</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="file">
       <property name="minimumSize">
        <size>
         <width>56</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#file:checked
{
border:none;
background-color: rgb(20, 153, 255);
color:white;
border-radius:2px;
}
QPushButton#file
{
border:none;
background-color: transparent;
color:black;
}
QPushButton#file:hover
{
background-color: rgb(20, 153, 255);
}</string>
       </property>
       <property name="text">
        <string>File</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="assitance">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
border:none;
	background-color: transparent;
}
QPushButton:hover
{
border:1px solid gray;
}
QPushButton:pressed
{
border:1px solid gray;
padding-top:2px;
padding-left:2px;
background-color: rgba(234, 234, 234,120);
}</string>
       </property>
       <property name="text">
        <string>File Assitance</string>
       </property>
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/res/talk/lnz.png</normaloff>:/res/talk/lnz.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="msg_all">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QWidget" name="condition" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>86</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>86</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton 
{
border:1px solid gray;
padding:1px;
border-radius:2px;
text-align: left;
	background-color: rgb(255, 255, 255);
}
QPushButton:hover
{
border:2px solid rgb(47, 141, 255);
padding:0px;
border-radius:2px;
}</string>
            </property>
            <property name="text">
             <string>All</string>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/talk/nnu.png</normaloff>:/res/talk/nnu.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="range">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>96</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit#range
{
border:1px solid gray;
border-radius:2px;
}
QLineEdit#range:hover
{
border:1px solid rgb(47, 141, 255);
border-radius:1px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ok">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#ok
{
border:none;
border-radius:2px;
}
QPushButton#ok:hover
{
border:1px solid rgb(173, 173, 173);
}
QPushButton#ok:pressed
{
border:1px solid rgb(173, 173, 173);
padding-top:1px;
padding-left:1px;
}</string>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottom" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="setting">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#setting
{
border:none;
}
QPushButton#setting:hover
{
border:none;
}
QPushButton#setting:pressed
{
border:none;
padding-top:2px;
padding-left:2px;
}</string>
            </property>
            <property name="text">
             <string>Setting</string>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/talk/setting.png</normaloff>:/res/talk/setting.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search">
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#search
{
border:none;
}
QPushButton#search:hover
{
border:none;
}
QPushButton#search:pressed
{
border:none;
padding-top:2px;
padding-left:2px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/talk/search.png</normaloff>:/res/talk/search.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="msgbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#msgbox
{
border:none;
}
QPushButton#msgbox:hover
{
border:none;
}
QPushButton#msgbox:pressed
{
border:none;
padding-top:2px;
padding-left:2px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/talk/qhj.png</normaloff>:/res/talk/qhj.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="calendar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#calendar
{
border:none;
}
QPushButton#calendar:hover
{
border:none;
}
QPushButton#calendar:pressed
{
border:none;
padding-top:2px;
padding-left:2px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/talk/sat.png</normaloff>:/res/talk/sat.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="first">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#first
{
border:none;
	border-image: url(:/res/talk/left.png);
}
QPushButton#first:pressed
{
border-top:2px solid transparent;
border-left:2px solid transparent;
	border-image: url(:/res/talk/left.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previous">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#previous
{
border:none;
	border-image: url(:/res/talk/left.png);
}
QPushButton#previous:pressed
{
border-top:2px solid transparent;
border-left:2px solid transparent;
	border-image: url(:/res/talk/left.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#next
{
border:none;
	border-image: url(:/res/talk/right.png);
}
QPushButton#next:pressed
{
border-top:2px solid transparent;
border-left:2px solid transparent;
	border-image: url(:/res/talk/right.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="last">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#last
{
border:none;
	border-image: url(:/res/talk/right.png);
}
QPushButton#last:pressed
{
border-top:2px solid transparent;
border-left:2px solid transparent;
	border-image: url(:/res/talk/right.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="msg_photo"/>
     <widget class="QListWidget" name="msg_file"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
